packages/thumbprint-react/components/Avatar/index.jsx

```diff
@@ -33,6 +34,29 @@ const getBadgeProps = ({ size, hasUnreadNotifications, isChecked, isOnline }) =>
 const shouldShowBadge = ({ size, hasUnreadNotifications, isChecked, isOnline }) =>
     size !== 'xsmall' && (hasUnreadNotifications || isChecked || isOnline);
 
+const BACKGROUND_COLOURS = [
+    tokens.tpColorIndigo600,
+    tokens.tpColorGreen600,
+    tokens.tpColorYellow600,
+    tokens.tpColorRed600,
+    tokens.tpColorPurple600,
+    tokens.tpColorBlue600,
+];
+
+const TEXT_COLOURS = [
+    tokens.tpColorIndigo100,
+    tokens.tpColorGreen100,
+    tokens.tpColorYellow100,
+    tokens.tpColorRed100,
+    tokens.tpColorPurple100,
+    tokens.tpColorBlue100,
+];
+
+const getBackgroundColour = initials =>
+    BACKGROUND_COLOURS[initials.charCodeAt(0) % BACKGROUND_COLOURS.length];
+
+const getTextColour = initials => TEXT_COLOURS[initials.charCodeAt(0) % TEXT_COLOURS.length];
```

Comment:
`initials` is not a required prop, so this will fail if it's not defined. I think that's why Jest is failing. Let's use `gray200` as a default background color in the off chance that neither an initial or an image are provided.

If you'd like, we can also use `warning` to add a warning for that case.

Reply:
Done